packages/next/build/webpack-config.ts

```diff
@@ -534,6 +534,27 @@ export default async function getBaseWebpackConfig(
     )
   }
 
+  const getPackagePath = (name: string, relativeToPath: string) => {
+    const packageJsonPath = require.resolve(`${name}/package.json`, {
+      paths: [relativeToPath],
+    })
+    return path.dirname(packageJsonPath)
```

Comment:
```suggestion
	// directory with a tailing slash
    return path.join(packageJsonPath, "../")
```

This is equal to `path.dirname(packageJsonPath) + path.sep`.

It gets parent directory with a tailing slash.

We want to include the tailing slash to not get false-positives like `node_modules/react-slider` when checking for `startsWith`

Reply:
Great catch, thanks. Updated.